src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -260,6 +287,23 @@ private void ValidateParameters()
             }
         }
 
+        private async Task ConfigurePerformanceLibrary(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            if (this.UsePerformanceLibraries)
+            {
+                if (this.CpuArchitecture == Architecture.Arm64)
+                {
+                    this.hplPerfLibraryInfo = "arm-performance-libraries_23.04.1_Ubuntu-22.04";
```

Comment:
what will happen if someone will run on ubuntu 20.04 or any future ones?

Reply:
If this type of change is only valid/tested for ubuntu22.04, could you please mention very clearly in our documentation that these options are only supported for this version and have not tested for other versions